
# PYTHON EXCEPTIONS

| Functions | Operations |
| :---: | :---: |
| **def safe_print_list(my_list=[], x=0)** | function that prints x elements of a list. |
| **def safe_print_integer(value)** | function that prints an integer with "{:d}".format(). |
| **def safe_print_list_integers(my_list=[], x=0)** |  function that prints the first x elements of a list and only integers. x can be bigger than the length of my_list - if itâ€™s the case, an exception is expected to occur |
| **def safe_print_division(a, b)** | function that divides 2 integers and prints the result. |
| **def raise_exception()** | function that raises a type exception. |
| **def raise_exception_msg(message="")** |  function that raises a name exception with a message. |
